.github/workflows/check-lambda-js.yml

```diff
@@ -36,28 +37,30 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '20.15.1'
+          node-version: '22.1.0'
```

Comment:
# Backend Error

1. It seems like [`licenses.csv`](https://github.com/csg-org/CompactConnect/blob/0978070151d9947bd27ec20da463fdaf787ab48d/backend/compact-connect/lambdas/python/common/tests/resources/licenses.csv) is not a valid CSV for uploading.
2. Despite that, the `/licenses/bulk-upload` endpoint returns a status 200 response (I think this is expected, since it should only be returning an upload URL), and the upload URL returns a status 204 response. It seems like this might be expected, but I don't see any validation errors in the DataEventTable (which I think I would expect).
3. I see the below log in AWS under the `Sandbox-PersistentStack-BulkUploadsBucketV1ParseOb` lambda.

```json
{
    "level": "INFO",
    "location": "process_bulk_upload_file:135",
    "message": "Invalid license in line 5 uploaded: {'licenseType': ['Must be one of: occupational therapist, occupational therapy assistant.']}",
    "timestamp": "2024-12-18 19:02:59,165+0000",
    "service": "service_undefined",
    "cold_start": true,
    "function_name": "Sandbox-PersistentStack-BulkUploadsBucketV1ParseOb-PdhaLidEQgJO",
    "function_memory_size": "1024",
    "function_arn": "arn:aws:lambda:us-east-1:010526247786:function:Sandbox-PersistentStack-BulkUploadsBucketV1ParseOb-PdhaLidEQgJO",
    "function_request_id": "35e5010e-96f0-43a5-9854-724ae6f9e0f4",
    "valid_data": {
        "licenseType": "speech-language pathologist",
        "status": "active",
        "givenName": "María",
        "middleName": "José",
        "familyName": "Carreño Quiñones",
        "dateOfIssuance": "2024-06-30",
        "dateOfRenewal": "2024-06-30",
        "dateOfExpiration": "2024-06-30"
    },
    "exception": "Traceback (most recent call last):\n  File \"/var/task/handlers/bulk_upload.py\", line 126, in process_bulk_upload_file\n    validated_license = schema.load({'compact': compact, 'jurisdiction': jurisdiction, **raw_license})\n                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/python/marshmallow/schema.py\", line 722, in load\n    return self._do_load(\n           ^^^^^^^^^^^^^^\n  File \"/opt/python/marshmallow/schema.py\", line 909, in _do_load\n    raise exc\nmarshmallow.exceptions.ValidationError: {'licenseType': ['Must be one of: occupational therapist, occupational therapy assistant.']}",
    "exception_name": "ValidationError",
    "stack_trace": {
        "type": "ValidationError",
        "value": "{'licenseType': ['Must be one of: occupational therapist, occupational therapy assistant.']}",
        "module": "marshmallow.exceptions",
        "frames": [
            {
                "file": "/var/task/handlers/bulk_upload.py",
                "line": 126,
                "function": "process_bulk_upload_file",
                "statement": "validated_license = schema.load({'compact': compact, 'jurisdiction': jurisdiction, **raw_license})"
            },
            {
                "file": "/opt/python/marshmallow/schema.py",
                "line": 722,
                "function": "load",
                "statement": "return self._do_load("
            },
            {
                "file": "/opt/python/marshmallow/schema.py",
                "line": 909,
                "function": "_do_load",
                "statement": "raise exc"
            }
        ]
    },
    "xray_trace_id": "1-67631c60-4d55644656a1a98f65211ae9"
}
```

Reply:
Ah- didn't know about that script. Helpful!